MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings

```diff
@@ -1,15 +1,15 @@
 /*
  * Update dialog buttons
  */
-"MSDistributeUpdateNow" = "Обновить сейчас";
-"MSDistributeAskMeInADay" = "Спросить завтра";
-"MSDistributeViewReleaseNotes" = "Просмотреть примечания к обновлению";
+"MSDistributeUpdateNow" = "Обновить";
+"MSDistributeAskMeInADay" = "Спросить через день";
```

Comment:
Better to leave it as it was

Reply:
This is copied form TouchDown. I know this doesn't look good but it needs to be addressed in a separate PR. @jwallra please update the file in `Localization/`